lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -70,7 +70,7 @@ class LAMPApp extends Script {
 
   addScriptHeader() {
     this.script = this.script.concat([
-      'sleep 3',
+      'READY=0; while ! `nc -z 127.0.0.1 3306` && [ $READY -lt 60 ]; do echo "Waiting for MySQL..."; READY=$((READY + 1)); sleep 1; done; if `nc -z 127.0.0.1 3306`; then echo "MySQL is ready."; else echo "MySQL failed to start!"; exit 1; fi;',
```

Comment:
@tizzo and I were talking the other day about a solution that may be better for the long term however it is more time to implement. This was a fast solution to resolve this issue in a fast but more reliable way. In any event, you have to admit that it is better than a hard coded value of 3 seconds.